config/crd/bases/telemetry.kyma-project.io_tracepipelines.yaml

```diff
@@ -342,22 +342,70 @@ spec:
               conditions:
                 description: An array of conditions describing the status of the pipeline.
                 items:
-                  description: TracePipelineCondition contains details for the current
-                    condition of this LogPipeline.
+                  description: "Condition contains details for one aspect of the current
+                    state of this API Resource. --- This struct is intended for direct
+                    use as an array at the field path .status.conditions.  For example,
+                    \n type FooStatus struct{ // Represents the observations of a
+                    foo's current state. // Known .status.conditions.type are: \"Available\",
+                    \"Progressing\", and \"Degraded\" // +patchMergeKey=type // +patchStrategy=merge
+                    // +listType=map // +listMapKey=type Conditions []metav1.Condition
+                    `json:\"conditions,omitempty\" patchStrategy:\"merge\" patchMergeKey:\"type\"
+                    protobuf:\"bytes,1,rep,name=conditions\"` \n // other fields }"
                   properties:
                     lastTransitionTime:
-                      description: Point in time the condition transitioned into a
-                        different state.
+                      description: lastTransitionTime is the last time the condition
+                        transitioned from one status to another. This should be when
+                        the underlying condition changed.  If that is not known, then
+                        using the time when the API field changed is acceptable.
                       format: date-time
                       type: string
+                    message:
+                      description: message is a human readable message indicating
+                        details about the transition. This may be an empty string.
+                      maxLength: 32768
+                      type: string
+                    observedGeneration:
+                      description: observedGeneration represents the .metadata.generation
+                        that the condition was set based upon. For instance, if .metadata.generation
```

Comment:
```suggestion
                        that the condition was set based upon. For example, if .metadata.generation
```

Reply:
Also the documentation here is auto-generated from the metav1 package https://github.com/kubernetes/apimachinery/blob/v0.29.1/pkg/apis/meta/v1/types.go#L1518